ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/filters/KsqlAuthorizationFilterTest.java

```diff
@@ -138,7 +139,8 @@ private ContainerRequest givenRequestContext(
         URI.create(path),
         method,
         securityContext,
-        mock(PropertiesDelegate.class)
+        mock(PropertiesDelegate.class),
+        null
```

Comment:
This change was applied a few days ago but seemingly reverted in https://github.com/confluentinc/ksql/pull/4580. Another datapoint for not merging PRs without seeing passing builds.